awx/main/tasks.py

```diff
@@ -3033,7 +3034,7 @@ def check_heartbeat(cls, node):
             'settings': {
                 "container_image": get_default_execution_environment().image,
                 "container_options": ['--user=root'],
-                "process_isolation": False if MODE == 'development' else True,
+                "process_isolation": True,
```

Comment:
@shanemcd Do we want to make this configurable at some point or are we thinking that execution nodes will always spin up containers locally? Guessing the latter, but wasn't 100% sure.